### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by LeslieAine**
feature branch to leave comments

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    <h1>TO-DO List</h1>
+    <div class="tasks-wrapper">
+      <div class="tasks-title"><label>Today's To Do</label><i class="fa-solid fa-arrows-rotate refresh-icon"></i></div>
+      <form class="todo-list">
+        <input class="input-text" type="text" id="input-task" placeholder="Add to your list...">
+        <button class="submit-btn" type="submit">Add</button>
+      </form>
+      <div class="todo-list-items"></div>
+
+      <p class="clear-all">Clear all completed</p>
```
Comment (Jolak5) [inline dist/index.html:20]: Hi, I suggest you wrap this section in a section tag to ensure a cleaner and well-separated code.

File: src/addRemove.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
+import { addToStorage, getFromStorage } from './updateLocalStorage.js';
```
Comment (Jolak5) [inline src/addRemove.js]: It is always better to have your dependencies in a single folder. I will suggest that you move them into a folder name "modules" for easy assessibility

**Review by Jolak5 — CHANGES_REQUESTED** [review]
You're almost there Leslie. Just a few additions and you're good to go

**Comment by LeslieAine** [comment]
Thank you Joseph for the reviews

**Pull request merged**